lib/rules/web/admin_console/4.2/monitoring_alerts.xyaml

```diff
@@ -116,4 +120,84 @@ status_specific_silence:
   element:
     selector:
       xpath: //div[@aria-label='Silences']//td[text()='<status>'][1]
-  
\ No newline at end of file
+
+set_alert_label_invalid:
```

Comment:
why it is called *_invalid

Reply:
because I didn't enable regular, but used regular

Reply:
yes, I realized from other cases